packages/core/src/components/accordion/accordion.template.ts

```diff
@@ -0,0 +1,57 @@
+import { Accordion, AccordionSection, AccordionDemoSection } from '@dso-toolkit/sources';
+import { html } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+
+export function accordionTemplate(
```

Comment:
De signature van TemplateFunctions kan niet meer dan één parameter hebben, anders kom ik met #1808 in de knoop.

Daarnaast denk ik dat `sections` een onderdeel van de `interface Accordion` moet zijn. Er mag toch nooit een ander type element in `<dso-accordion>`.

Reply:
Ik twijfel steeds wat ik vind van het plaatsen van wat eigenlijk "child HTMLElementen" zijn als 'property' op de interface. Omdat het bij de afnemer niet te typeren valt met typescript, en de andere props/methods etc wel.

van de tweede parameter ben ik nu af.

Reply:
De interfaces zijn niet voor afnemers, maar voor ons: Een eigenschap van accordion is dat er sections in staan. Ik ga kijken.

Reply:
Die staan er nu als eigenschap 😄